src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -6082,6 +6082,7 @@ public enum DynamicallyAccessedMemberTypes
         NonPublicProperties = 1024,
         PublicEvents = 2048,
         NonPublicEvents = 4096,
+        Interfaces = 8192,
```

Comment:
Is there anywhere we should be using this now that we have it?

Reply:
Yes. But that will require a few changes throughout the repo, and we will need the linker to start respecting this new value. I assume this is to unblock the linker work.

Reply:
Exactly - we need to flow this to linker, teach linker about it and then we can apply the runtime/framework changes.

Reply:
Basically tracked by https://github.com/mono/linker/issues/1731.